plugin-common/src/main/kotlin/com/gradleup/gr8/Gr8Configurator.kt

```diff
@@ -23,9 +27,17 @@ open class Gr8Configurator(
   private var proguardFiles = mutableListOf<Any>()
   private var stripGradleApi: Property<Boolean> = project.objects.property(Boolean::class.java)
   private var excludes: ListProperty<String> = project.objects.listProperty(String::class.java)
+  private val javaLauncher: Property<JavaLauncher> = project.objects.property(JavaLauncher::class.java)
```

Comment:
I'm not sure why the other props are `var` here?

Reply:
Most likely an oversight. These should be `val` since they're mutable `Properties` already